crud_file.c

```diff
@@ -0,0 +1,217 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#define MAX_SIZE 100
+
+// User schema structure
+struct User {
+    char id[MAX_SIZE];
+    char name[MAX_SIZE];
+    int age;
+};
+
+// Helper function to check if a user exists
+int userExists(const char *id) {
+    FILE *fp = fopen("users.txt", "r");
+    if (!fp) return 0;
+
+    char buffer[MAX_SIZE];
+    struct User user;
+
+    while (fgets(buffer, sizeof(buffer), fp)) {
+        if (sscanf(buffer, "%[^,],%[^,],%d", user.id, user.name, &user.age) == 3) {
+            if (strcmp(user.id, id) == 0) {
```

Comment:
Do not use inbuilt functions.
Make your own functions and use them and this comment is for all places where you have used inbuilt functions
Check and correct them